src/raygun.rum/index.js

```diff
@@ -897,6 +918,13 @@ var raygunRumFactory = function(window, $, Raygun) {
     // =                                                                              =
     // ================================================================================
 
+    function resumeCollectingMetricsFromNow() {
```

Comment:
Is the `FromNow` portion needed here? I'd imagine that from a usage perspective, it would resume collecting at the point you invoke the method,

Reply:
Good point. I've rename the method removing the `FromNow` portion